content/geometry/General.adoc

```diff
@@ -4,19 +4,41 @@
 The unit used for object dimensions is meters.
 
 == Coordinate systems
-The local coordinate system is a right-handed coordinate system according to
-ISO 8855 with the axes pointing to the directions mentioned below. For a non-rotated
-coordinate system, the following applies:
+
+ASAM OpenMATERIAL uses the following types of coordinate systems:
+
+* World coordinate system, see <<sec-World-coordinate-system>>
+* Local coordinate system, see <<sec-Local-coordinate-system>>
+
+Both coordinate systems consist of three orthogonal directions associated with
+X, Y, and Z axes, and a coordinate origin where axes meet. The coordinate
+systems are right-handed Cartesian coordinate systems according to ISO 8855. For
+a non-rotated coordinate system, the following applies:
 
 * Forward matches x-axis
 * Left matches y-axis
 * Up matches z-axis
 
-NOTE: TODO: Add images similar to OpenDRIVE? See
-https://publications.pages.asam.net/standards/ASAM_OpenDRIVE/ASAM_OpenDRIVE_Specification/latest/specification/08_coordinate_systems/08_04_local_coordinate_system.html
+The positive rotation is assumed to be counter-clockwise ("right-hand rule").
+Orientation of objects is expressed extrinsically by the yaw, pitch, and roll
+angles derived from the Euler rotation sequence in the order Z-axis, then
+Y-axis, then X-axis.
+
+TODO: Add image of right-handed coordinate system with rotation directions
+
```

Comment:
We are currently waiting for access to OpenDRIVE repo, because we want to reuse the existing images (https://publications.pages.asam.net/standards/ASAM_OpenDRIVE/ASAM_OpenDRIVE_Specification/latest/specification/08_coordinate_systems/08_01_introduction.html)

Reply:
resolving conversation, because we now have access to the source files